web/src/settings_notifications.ts

```diff
@@ -283,6 +282,7 @@ export function set_up(settings_panel: SettingsPanel): void {
             return;
         }
 
+        const setting_name = user_settings_schema.keyof().parse($input_elem.attr("name"));
```

Comment:
This seems a good solution as we anyways handle the notification batching setting as a special case and is better to not modify or create a new zod schema to include the "_minutes" field to handle this.